doc/tutorials/intermediate/serve.md

```diff
@@ -0,0 +1,256 @@
+# Serve Apps
+
+In this tutorial you will learn more advanced techniques to serve Panel apps:
+
+- serve a multi page app by providing a list of files or [*globs*](https://en.wikipedia.org/wiki/Glob_(programming)).
+- list the configuration options of `panel serve` by adding the flag `--help`.
+
+:::{admonition} Note
+This guide builds upon the [Serve Panel Apps (beginner)](../beginner/) tutorial.
+:::
+
+## Serve a multi-page app
+
+You can serve an app with multiple pages just by providing the file paths to `panel serve`.
+
+Copy the code below into a file named `app.py` and save the file
+
+```python
+import panel as pn
+
+pn.extension()
+
+pn.panel("Hello World").servable()
+```
+
+Copy the 2 code cells below into a clean notebook named `app2.ipynb`.
+
+```python
+import panel as pn
+
+pn.extension()
+```
+
+```python
+pn.panel("Hello Notebook World").servable()
+```
+
+Run the cells and save the notebook if you have not already done it.
+
+Run `panel serve app.py app2.ipynb --autoreload`.
+
+It should look like
+
+```bash
+$ panel serve app.py app2.ipynb --autoreload
+2024-01-17 21:14:43,502 Starting Bokeh server version 3.3.3 (running on Tornado 6.4)
+2024-01-17 21:14:43,503 User authentication hooks NOT provided (default user enabled)
+2024-01-17 21:14:43,506 Bokeh app running at: http://localhost:5006/app
+2024-01-17 21:14:43,506 Bokeh app running at: http://localhost:5006/app2
+2024-01-17 21:14:43,506 Starting Bokeh server with process id: 39832
+```
+
+Open [http://localhost:5006](http://localhost:5006), [http://localhost:5006/app](http://localhost:5006/app) and [http://localhost:5006/app2](http://localhost:5006/app2).
+
+It will look something like
+
+![Panel serve multi page app](../../_static/images/panel-serve-multipage-app.png)
+
+:::{admonition} Note
+You can also use one or more [*globs*](https://en.wikipedia.org/wiki/Glob_(programming)) to serve a long or dynamic list of apps. For example `apps/*`, `pages/*.py`, `examples/*.ipynb` or `docs/*.md`.
+:::
+
+## Configure `panel serve`
+
+The command **`panel serve` is highly configurable**.
+
+Run `panel serve --help` in a terminal.
+
+It should look like the below.
+
+```bash
+$ panel serve --help
+usage: panel serve [-h] [--port PORT] [--address ADDRESS] [--unix-socket UNIX-SOCKET] [--log-level LOG-LEVEL] [--log-format LOG-FORMAT]
```

Comment:
Does this need to be in the tutorial, or can this be linked? I fear that it'll be outdated in the future.

Reply:
Thx. It will be outdated in the future. We have the same output in another part of our docs too.

I believe its important to include. And that its a bit outdated does not matter to users.

Reply:
When this has been merged, feel free to open a PR removing or changing it. If you get that approved its fine with me.

But right now this is the PR I believe in 😄